### DISCUSSION TITLE
JSON stats output

### DISCUSSION THREAD

**Discussion by kaspar030** [ccache/ccache#1379] [Ideas]
I'd be quite interested in having versions of `--print-stats` and `--show-log-stats` that directly output to JSON.

Would that be interesting to ccache, as in, if I'd contribute an implementation, would it have a chance to get merged?

**Comment by kaspar030** [comment]
Well, I gave it a shot in https://github.com/ccache/ccache/pull/1380.

**Comment by jrosdahl** [comment]
Yes, that would welcome.
